tools/scripts/get-user-token.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+username=$1
+
+if [ -z "${username}" ]; then
+    echo "Usage ${0} my-username"
+fi
+
+podman --remote exec --user=root -it docker-compose_django_1 wisdom-manage createtoken --username ${username} --duration 6000000 --token-name my-dev-token
```

Comment:
It might have been nice(r) to have done something like this:

https://github.com/ansible/ansible-ai-connect-service/blob/main/Makefile#L5-L10